.github/workflows/unbundled-lint-and-test.yaml

```diff
@@ -160,15 +160,21 @@ jobs:
           rustup show
           rustup component add rustfmt clippy
 
-      - name: Build the project
+      - name: Build the project without Zenoh & vsomeip transports
         working-directory: ${{github.workspace}}
-        run: cargo build --no-default-features
+        run: cargo build
+      - name: cargo clippy without Zenoh & vsomeip transports
+        working-directory: ${{github.workspace}}
+        run: cargo clippy --all-targets -- -W warnings -D warnings
+      - name: Build the project with Zenoh & vsomeip transports (and thus streamer references)
+        working-directory: ${{github.workspace}}
+        run: cargo build --features enable-vsomeip-transport,bundled-vsomeip,enable-zenoh-transport
+      - name: cargo clippy with Zenoh & vsomeip transports (and thus streamer references)
+        working-directory: ${{github.workspace}}
+        run: cargo clippy --features enable-vsomeip-transport,bundled-vsomeip,enable-zenoh-transport --all-targets -- -W warnings -D warnings
```

Comment:
Is there any difference anymore between this and `bundled-lint-and-test.yaml`?

Reply:
I... think you're right. I should not have included the `bundled-vsomeip` flag here!

Reply:
Done